### ISSUE TITLE
System Report -> Modules should not list plugins

### ISSUE THREAD

**Issue by nettum** [craftcms/cms#3513]
### Description
"Utilities" -> "System Report" -> "Modules" should not list plugins?
Right now all installed plugins will be listed both under "Plugins" and under "Modules":
![image](https://user-images.githubusercontent.com/798156/49375308-38d1bf00-f704-11e8-9a32-dc98b7a5e9eb.png)
![image](https://user-images.githubusercontent.com/798156/49375332-48e99e80-f704-11e8-8103-124c45e987a8.png)

Ref. #3490

### Steps to reproduce

1. Update craft to 3.0.33
2. Goto "Utilities" -> "System Report" and look under "Modules"

### Additional info

- Craft version: 3.0.33
- PHP version: N/A
- Database driver & version: N/A
- Plugins & versions: N/A

**Comment by brandonkelly** [comment]
Yep, fixed that in 7016ee3e4b1230f94a9096a60d8197f5e999c0b6 for the next release.
